_test/search_test.rb

```diff
@@ -9,5 +9,10 @@ def test_index_built
       assert(File.exist?(File.join(SiteBuilder::BUILD_DIR, 'index.json')),
         "Serialized lunr.js index doesn't exist")
     end
+
+    def test_url_to_doc_map_built
+      assert(File.exist?(File.join(SiteBuilder::BUILD_DIR, 'url_to_doc.json')),
+        "Serialized URL-to-doc info index doesn't exist")
+    end
```

Comment:
We should actually check the contents, e.g. that it has `url` and `title` on every entry.